venv/PlottingClass.py

```diff
@@ -0,0 +1,36 @@
+import pandas as pd
```

Comment:
Store all your 3rd party libraries in [requirements.txt](https://note.nkmk.me/en/python-pip-install-requirements/)